shard-operator/src/test/resources/application.properties

```diff
@@ -1,2 +1,4 @@
 quarkus.scheduler.enabled=false
-quarkus.container-image.build=false
\ No newline at end of file
+quarkus.container-image.build=false
+
+quarkus.oidc-client.enabled=false
```

Comment:
Here I disable sso for unit test

Reply:
Shouldn't it be `quarkus.oidc-client.client-enabled=false`?
As shown in https://quarkus.io/guides/security-openid-connect-client#use-oidcclients